core/write_test.go

```diff
@@ -252,44 +252,11 @@ func TestFindFieldIndex_tagOverwriteFieldName(t *testing.T) {
 	}
 }
 
-type testSettable struct {
-	Value string
-}
-
-type testSettableStruct struct {
```

Comment:
The refactor lost the test on using custom types as a struct field, so I would add something like this back in:
```
type testSettableStruct struct {
    Settable testFieldSettable `survey:"values"`
}
```
Then in the `TestWriteWithFieldSettable` func add something like:
```
testSetStruct := testSettableStruct{}		
err = WriteAnswer(&testSetStruct, "values", "stringVal1")		
assert.Nil(t, err)		
assert.Equal(t, testSetStruct.Settable.Value, "stringVal1")
```